extensions/systemd/CMakeLists.txt

```diff
@@ -17,12 +17,19 @@
 # under the License.
 #
 
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
+  option(ENABLE_SYSTEMD "Enables the systemd extension." ON)
+endif()
```

Comment:
I would keep this part in the root `CMakeLists.txt` file, together with the other `option(...)` commands

Reply:
agreed, at first I wanted to move (and did move) everything including `option(...)`s but then moved them back, this one was duplicated here by accident, removed it